packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -241,6 +243,9 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
           this.distanceFromTop - container.scrollTop >= this._getStickyDistanceFromTopForFooter(container, footerStickyContainer);
       }
 
+      if (this.nonStickyContent && !(this.state.isStickyTop || this.state.isStickyBottom)) {
+        this._nonStickyPlaceHolderScrollWidth = this.nonStickyContent.scrollWidth;
+      }
```

Comment:
Do we need to update this on every onScroll call? Seems redundant.

Reply:
Probably, you would want to do this only when current state is non-sticky and new state is sticky.

---
In reply to: [253080989](https://github.com/OfficeDev/office-ui-fabric-react/pull/7855#discussion_r253080989) [](ancestors = 253080989)